api/config/config_test.json

```diff
@@ -90,7 +90,7 @@
     "enableVpcServicePerimeter": true,
     "enableNewAccountCreation": true,
     "enableRdrExport": true,
-    "enableBillingLockout": false,
```

Comment:
is this intentional?

Reply:
yeah, I figure we can let it go on test now. It seems to work from my testing.